third_party/silabs/efr32_sdk.gni

```diff
@@ -143,10 +158,16 @@ template("efr32_sdk") {
       "${efr32_sdk_root}/platform/security/sl_component/se_manager/src/",
       "${efr32_sdk_root}/util/third_party/freertos/cmsis/Include",
       "${efr32_sdk_root}/util/third_party/freertos/kernel/include",
-      "${sdk_support_root}/matter/efr32/${silabs_family}/${silabs_board}/config",
-      "${sdk_support_root}/matter/efr32/${silabs_family}/${silabs_board}/autogen",
+      "${efr32_sdk_root}/platform/driver/debug/inc",
+      "${silabs_gen_folder}/config",
+      "${silabs_gen_folder}/autogen",
     ]
 
+    if (generate_slc || use_slc_files) {
```

Comment:
what is the reason here for only with slc ? seems like it was included for BRD4318A and BRD2704A before